requirements.txt

```diff
@@ -1,8 +1,9 @@
 abjad==3.4
+albumentations==1.3.0
```

Comment:
Why is the requirements txt changed? Wasn't illuminations already present?

Reply:
idk